libvirt/tests/src/daemon/crash_regression.py

```
@@ -40,8 +40,11 @@ def run_shutdown_console(params, libvirtd, vm):
     """
     Start a vm with console connected and then shut it down.
     """
-    vm.start(autoconsole=True)
+    vm_xml = VMXML.new_from_inactive_dumpxml(vm.name)
+    vm_xml_backup = vm_xml.copy()
+    libvirt_vmxml.modify_vm_device(vmxml, 'interface', {'driver': None})
```

Ruff (0.14.5)

Undefined name `libvirt_vmxml`

(F821)